relentless/optimize/method.py

```diff
@@ -145,15 +153,23 @@ class LineSearch:
 
         \mathbf{d} = \mathbf{x}_{end} - \mathbf{x}_{start}
 
-        \hat{\mathbf{d}} = \frac{\mathbf{d}}{\lVert\mathbf{d}\rVert}
+        \mathbf{\hat{d}} = \frac{\mathbf{d}}{\lVert\mathbf{d}\rVert}
 
     Then, a "target" value :math:`t` is defined as:
 
     .. math::
 
-        t = -\hat{\mathbf{d}} \cdot \nabla{f\left(\mathbf{x}\right)}
+        \nabla f\left(\mathbf{\hat{x}}\right) = \left[{b_1}^2 \frac{\partial f}{\partial x_1},
```

Comment:
Can we use a different symbol for the scale parameters that attaches a little more meaning? Maybe capital X? I also think we shouldn't use `\hat` to indicate the scaled variables, since it also is used to indicate the unit vector. The result of scaling the variables is not necessarily unity. I would maybe suggest `y = x/X`? Then you can also write like `\nabla_y` and `\nabla_x` pretty cleanly.